onnxruntime/test/framework/float_8_test.cc

```diff
@@ -52,6 +52,8 @@ TEST(Float8_Tests, NanE4M3FN) {
   // 0x7FC00000 is the value used by numpy.
   EXPECT_EQ(onnxruntime::Float8E4M3FN((float_bits{0x7FC00000}).val).val, static_cast<uint8_t>(0x7F));
   EXPECT_EQ(onnxruntime::Float8E4M3FN((float_bits{0xFFC00000}).val).val, static_cast<uint8_t>(0xFF));
+  // small negative values should round to negative zero
+  EXPECT_EQ(onnxruntime::Float8E4M3FN(-0.00000001).ToFloat(), -0.0f);
```

Comment:
Some doubts about this: EXPECT_EQ(-0.0f, 0.0f) is true or false? That may be better to write that test the same way it is written just above.

Reply:
right, EXPECT_EQ(-0.0f, 0.0f) is true.
It may be good to leave it as documentation of intent, but
internal representation as 0x80 should be tested.

Reply:
updated tests with internal representation and added "f" suffix which should hopefully fix warnings; I hope this is OK now.